packages/docs/src/components/ButtonGroup/sections.tsx

```diff
@@ -30,72 +31,196 @@ export const variants = [
         <Button>Button 2</Button>
         <Button>Button 3</Button>
       </ButtonGroup>
-    )
-  }
+    ),
+  },
 ];
 
-export const colors = [
+const contained = [
   {
-    title: 'Primary',
+    title: 'Contained / Primary',
     component: (
       <ButtonGroup color="primary" variant="contained" size="medium">
         <Button>Button 1</Button>
         <Button>Button 2</Button>
         <Button>Button 3</Button>
       </ButtonGroup>
-    )
+    ),
   },
   {
-    title: 'Secondary',
+    title: 'Contained / Secondary',
     component: (
       <ButtonGroup color="secondary" variant="contained" size="medium">
         <Button>Button 1</Button>
         <Button>Button 2</Button>
         <Button>Button 3</Button>
       </ButtonGroup>
-    )
+    ),
   },
   {
-    title: 'Default',
+    title: 'Contained / Default',
     component: (
       <ButtonGroup color="default" variant="contained" size="medium">
         <Button>Button 1</Button>
         <Button>Button 2</Button>
         <Button>Button 3</Button>
       </ButtonGroup>
```

Comment:
We could extract this in a component and in each variant pass different props as needed, something like:
```
const SampleButtonGroup = (props) => <ButtonGroup {...props} >... </ButtonGroup>

// and later

title: 'Default',
component: (<SampleButtonGroup color="default" variant="contained" size="medium" />)
```